dtmi/vecow/AIC-110

```diff
@@ -0,0 +1,13 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:VECOW:aic_110;1",
+    "@type": "Interface",
+    "displayName": "VECOW-AIC-110",
+    "contents": [
+        {
+            "@type": "Component",
+            "name": "LinuxDeviceInfo1",
+            "schema": "dtmi:Synnex:LinuxDeviceInfo;1"
```

Comment:
@chrishuang19790528 -- same request with this PR, since @WayneHou118 first created the `dtmi/vecow/*` namespace can we have them sign-off after the file name issue is fixed?

Reply:
yes, we are the same team at Vecow